### PR TITLE
NBSNEBIUS-89: [Tasks] extract scheduleInterval and maxTasksInflight into TaskSchedule struct; rename tasks/storage.schedule to scheduleState

### PR DESCRIPTION
**Pull request by SvartMetal**
In next PRs "cron tab" parameters will be added (and encapsulated inside TaskSchedule struct along with ScheduleInterval and MaxTasksInflight)

### CODE REVIEW

**Review by jkuradobery — COMMENTED** [review]
I would rather rename Introduce TaskSchedule to Extract scheduleInterval and maxTasksInflight into TaskSchedule struct. This way it would be more clear that you do not introduce new functionality, just refactor the old one.

**Comment by jkuradobery** [comment]
> In next PRs "cron tab" parameters will be encapsulated inside TaskSchedule struct

Is it really required?

File: cloud/tasks/storage/storage_ydb_impl.go [hunk 1]
```diff
@@ -686,14 +685,14 @@ func (s *storageYDB) createRegularTasks(
 	defer res.Close()
 
 	found := false
-	sch := schedule{}
+	schState := scheduleState{}
```
Comment (jkuradobery) [inline cloud/tasks/storage/storage_ydb_impl.go:688]: Why not just create a ScheduleState variable, schState is understandable given the context, yet I think that we should measure up to higher expectations regarding the readability of the code.

**Review by jkuradobery — COMMENTED** [review]
Otherwise the code seems to be OK. I see two refactorings here, and the first is a direct cause of the second. If you need it merged ASAP, DM me.

**Review by jkuradobery — APPROVED** [review]

**Review by BarkovBG — APPROVED** [review]

**Pull request merged**